hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -51,11 +57,79 @@ class HassioIngressView extends LitElement {
       clearInterval(this._sessionKeepAlive);
       this._sessionKeepAlive = undefined;
     }
+
+    this._resolveIngressStatus = undefined;
+    this._resolveIngressTime = undefined;
   }
 
-  protected render(): TemplateResult {
+  public connectedCallback() {
+    super.connectedCallback();
+  }
+
+  private async _resolveURL(addonSlug: string): Promise<void> {
+    await this._fetchData(addonSlug);
```

Comment:
Let `_fetchData` return the addon (maybe rename to `fetchAddon`)
```suggestion
    this._addon = await this._fetchData(addonSlug);
```